filament/include/filament/IndirectLight.h

```diff
@@ -279,6 +279,11 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
      * Returns the rigid-body transformation applied to the IBL.
      */
     const math::mat3f& getRotation() const noexcept;
+
+    /**
+     * Returns an estimation of the indirect light's direction
```

Comment:
You should add more detail: explain that this direction can be used as the direction of a directional light, etc.

Reply:
Ok, I'll add more detail when I add the color/intensity api.